packages/patternfly-4/react-docs/gatsby-config.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
             { section: 'charts' },
             { section: 'components' },
             { section: 'demos' },
-            { section: 'experimental' },
+            { section: 'beta' },
```

Comment:
This should go away - all beta components will be listed with the others - just noted differently in the documentation.  If you'd rather do that in a separate PR (or have somebody else make that change, it's fine with me).